make_html.py

```diff
@@ -17,6 +20,17 @@
 
 from htmlify_comment import htmlify_comment
 
+@functools.cache
+def github_labels(pr):
+    url = f'https://api.github.com/repos/leanprover-community/mathlib4/pulls/{pr}'
+    response = requests.get(url)
+    json_response = response.json()
+    if 'labels' not in json_response:
+        logging.error(f'curl of github failed for {pr=}, API rate limit exceeded?')
+        return []
+    labels = [{'name': label['name'], 'color': label['color']} for label in json_response['labels']]
+    return labels
```

Comment:
It probably makes sense to use the python github API here, like we do in https://github.com/leanprover-community/mathlib4/blob/master/scripts/make_port_status.py

Reply:
done in very similar ways to what is done in make_port_status.py. I guess the CI just needs to somehow make sure to have a proper token in the `github-token` file